pkg/apis/edge/v1alpha1/edge-placement.go

```diff
@@ -73,32 +73,29 @@ type EdgePlacement struct {
 }
 
 // EdgePlacementSpec holds a desired binding between (a) a set of Locations and
-// (b) a set of objects to downsync and a way of identifying objects to upsync.
+// (b) a way of identifying objects to downsync and a way of identifying objects to upsync.
 type EdgePlacementSpec struct {
 	// `locationSelectors` identifies the relevant Location objects in terms of their labels.
 	// A Location is relevant if and only if it passes any of the LabelSelectors in this field.
 	LocationSelectors []metav1.LabelSelector `json:"locationSelectors,omitempty"`
 
-	// `namespaceSelector` identifies the relevant Namespace objects in terms of their labels.
-	NamespaceSelector metav1.LabelSelector `json:"namespaceSelector,omitempty"`
-
-	// `nonNamespacedObjects` defines the non-namespaced objects to bind with the selected Locations.
-	// +optional
-	NonNamespacedObjects []NonNamespacedObjectReferenceSet `json:"nonNamespacedObjects,omitempty"`
+	// `downsync` selects the objects to bind with the selected Locations for downsync.
+	// An object is selected if it matches at least one member of this list.
+	Downsync []DownsyncObjectTest `json:"downsync,omitempty"`
```

Comment:
```suggestion
	Downsync []DownsyncObjectCriteria `json:"downsync,omitempty"`
```

Reply:
What is the reason for suggesting "Criteria" instead of "Test"?

Reply:
I made the connection to QA (testing) rather than a test for a condition.
when you replicate something you do so based on a criteria. The term test is correct, but I don't think it is helpful to understand the API

Reply:
I have updated the comment on that type, hopefully it is clearer now.  Is this good enough for you?